### ISSUE TITLE
Problems with the config

### ISSUE THREAD

**Issue by schyan10** [Fabrizz/MMM-OnSpotify#72]
Hello everyone, I can install the module and the authentication also works. As soon as I want to adjust the config for my interest, the MM goes into error. How did you customize your Conifig so that you can display it?

Thank you for your support

**Comment by Fabrizz** [comment]
Can you share your MMM-OnSpotify config?

**Comment by Fabrizz** [comment]
Remember to add the comma after the default config:
```js
        {
            module: "MMM-OnSpotify",
            position: "bottom_right",
            config: {
                clientID: "x",
                clientSecret: "x",
                accessToken: "x",
                refreshToken: "x-x-x-x-x-x-x", // <--- Add a comma here
                /* Config example */
                experimentalCSSOverridesForMM2: [
			["--color-text-dimmed", "palette_vibrantlight",],
			["--ONSP-OVERRIDES-ICONS-COLOR", "palette_vibrantlight"],
		],
            }
        },
```

**Comment by schyan10** [comment]
Yes, I was able to find the error, but I have no connection to Spotify...
 
![Screenshot 2024-05-17 153009](https://github.com/Fabrizz/MMM-OnSpotify/assets/165216925/80f8ee30-b381-4e45-8a2a-1075111db811)
![WhatsApp Bild 2024-05-17 um 15 29 33_023e7286](https://github.com/Fabrizz/MMM-OnSpotify/assets/165216925/1cc4837a-aa45-4510-b9ce-da09806a34ac)

**Comment by Fabrizz** [comment]
Can you test removing completly the module and installing this version:

```bash
git clone --branch v4.0.0 https://github.com/Fabrizz/MMM-OnSpotify.git
```

**Comment by schyan10** [comment]
Yes, I'd like to try it out, but which command can I use to delete the module?

**Comment by schyan10** [comment]
I was able to delete the module through research, I have now used your version and used the existing client, clietID, acesstoken and refreshtoken. However, I still have no connection to Spotify...

**Comment by Fabrizz** [comment]
Thats really strange, is your account premium right?

**Comment by schyan10** [comment]
Yes, I have a Premium Family account

**Comment by dientuki** [comment]
Maybe you are having conflict with another module, disable or remove everything but MMM-OnSpotify and check again
